src/state/feed-feedback.tsx

```diff
@@ -1,6 +1,8 @@
 import React from 'react'
 import {AppState, AppStateStatus} from 'react-native'
 import {AppBskyFeedDefs} from '@atproto/api'
+// @ts-ignore
```

Comment:
sorry

Reply:
Maybe:

```typescript
// store in a .d.ts file
declare module 'bloomfilter' {
  export class BloomFilter {
    buckets: Int32Array;

    /**
     * Create a new empty bloom filter of size m with hashes k or
     * provide buckets as a number[] or Int32Array to deserialize a bloom filter
     * @param mOrBucketsArray number of bits (will be rounded up to nearest 32), or buckets
     * to deserialize into a filled bloomfilter
     * @param k number of hashes
     */
    constructor(m: number | number[] | Int32Array, k: number);

    /**
     * Add a value to a bloom filter
     * @param value
     */
    add(value: any): void;

    /**
     * Test whether a value exists in a bloom filter. (False positives are
     * possible, false negatives are not.)
     */
    test(value: any): boolean;

    /**
     * Estimated cardinality.
     */
    size(): number;
  }
}
```

Based on https://www.npmjs.com/package/@types/bloomfilter#additional-details and https://github.com/jasondavies/bloomfilter.js/blob/master/bloomfilter.js#L72-L77

Reply:
can't we just use @types/bloomfilter?

Reply:
You can, but then you miss `size(): number`.

Reply:
Personally I think https://github.com/bluesky-social/social-app/pull/7145 is right, and instead you'll want to "just" create a local typescript file, even, porting the 7 year old code. Maintain it locally.